frontend/common/src/constants/localizedConstants.tsx

```diff
@@ -113,6 +113,45 @@ export const getEducationRequirement = (
     `Invalid Education Requirement '${educationRequirementId}'`,
   );
 
+export const employmentDurationShort = defineMessages({
+  term: {
```

Comment:
I know this is what was done above with `eductionRequirements` but I feel like these keys should actually be constants that could be used here and in `SearchForm`.  I'm not totally sure where the constants would be.  Maybe right here in this file beside their localization?

Reply:
I agree that this seemed like an odd one out. Will convert this :)